libs/librepcb/common/graphics/graphicsview.cpp

```diff
@@ -203,6 +203,39 @@ void GraphicsView::zoomAnimationValueChanged(const QVariant& value) noexcept {
  *  Inherited from QGraphicsView
  ******************************************************************************/
 
+bool GraphicsView::event(QEvent* event) {
```

Comment:
Wouldn't it be possible to handle `QNativeGestureEvent` in the `eventFilter()` method where we already handle every other event? The event handling of `GraphicsView` is special since we don't handle our own events, but those of the underlying `GraphicsScene` object. This way we also get scene events (with scene coordinates) in addition to every other event. So if possible, I would like to handle `QNativeGestureEvent` in the `eventFilter()` method too (to reduce complexity and for consistency).

Reply:
It is possible. It originally wasn't sending the event because my `grabGesture()` was getting lost after `setUseOpenGl()` reset the viewport. Now that I figured that out, I receive the gesture events in the `eventFilter()`.